students/zima7/hw-3/answers_1.md

```diff
@@ -0,0 +1,20 @@
+1. console.log(a) = undefined --> variable not declared in function scope
```

Comment:
Please be aware, that moments when you declare a function and use it are different.

Would you update your answer based on that?

Reply:
A variable declared with let is visible only within the block in which it is declared.